index.html

```diff
@@ -9,16 +9,48 @@
 </head>
 <body>
     <header>
-
+        <nav>
+<h2>Awesome Books</h2>
+<ul class="navbar-ulist">
+    <li><a id="0" href="#list" class="navbar-list active">List</a></li>
+    <li><a id="1" href="#add-new" class="navbar-list">Add new</a></li>
+    <li><a id="2" href="#contact" class="navbar-list">Contact</a></li>
+</ul>
+        </nav>
+        <p class="date">Date</p>
     </header>
     <main>
+<section class="list active" id="list">
+    <h2>All awesome books</h2>
+    <ul class="book-list">
+    </ul>
+</section>
+<section class="add-new" id="add-new">
+    <h2>Add anew book</h2>
+    <form class="form">
+        <input type="text" name="title" id="title" placeholder="Title">
+        <input type="text" name="author" id="author" placeholder="Author">
+        <button type="submit" id="btn">Add</button>
```

Comment:
Great job @shella12 🤝

- [ ] [Optional] You may kindly implement the functionality of the add button and the delete button.